src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <header class="search-container">
+      <form
+        class="search-bar top-search-bar"
+        data-qa="big"
+      >
+        <input
+          class="search-bar-input search-bar-input--top"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
```

Comment:
Consider adding an empty line between the `header` tag and the `form` tag to add some 'air' and simplify reading, as they are sibling elements. However, do not add an empty line between the `form` and its child `input` as they are parent and child elements.